### DISCUSSION TITLE
i am clicker people who dont know code/ how do i change model that i want to use

### DISCUSSION THREAD

**Discussion by maloadjav** [openai/whisper#1152] [Q&A]
this is what i used  now..  because i dont have ...decent  or  enough  cpu and gpu 

(https://colab.research.google.com/github/ANonEntity/WhisperWithVAD/blob/main/WhisperWithVAD.ipynb#scrollTo=sos9vsxPkIN7)

you see ... i click and click  and i wait for the srt file .

i  know a little bit ..  this program use  ordinary  model  ...

so if i want  the program to use other  model  such as 

https://huggingface.co/cl-tohoku/bert-large-japanese-char

===  what do i need to do  ??? insert some command line ????  where  can i do that  ??? 
what is command line ???

**Comment by taucontrib** [comment]
In your notebook, click _show code_ in the _run whisper_ block, then change the line 
`model = whisper.load_model(model_size)`
to
`model = whisper.load_model('small')`
and if it's still to big, go for any of the models from https://github.com/openai/whisper#available-models-and-languages

**Comment by maloadjav** [comment]
thank you so much

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
